src/java.desktop/windows/native/libawt/windows/awt_TextComponent.cpp

```diff
@@ -53,12 +53,14 @@ struct EnableEditingStruct {
  * AwtTextComponent fields
  */
 
+jmethodID AwtTextComponent::canAccessClipboardMID;
+AwtTextComponent::OleCallback AwtTextComponent::sm_oleCallback;
+WNDPROC AwtTextComponent::sm_pDefWindowProc = NULL;
+
```

Comment:
Did the compiler complain here? I'm fine with the change, just wanted to know the reason.

Reply:
the latter two are inside an extern "C" block, meaning their initial C++ linkage (by virtue of them being static class members) conflicts with the now C linkage, also the comment there states the AwtComponent fields are supposed to be set here, and I have no idea why this was not done, so I moved them all to be under that comment

Reply:
The question seem answered. Please resolve the discussion.